card/spec/lib/card/reference_spec.rb

```diff
@@ -8,7 +8,7 @@
 
   describe "references on hard templated cards should get updated" do
     it "on structuree creation" do
-      Card.create! :name=>"JoeForm", :type=>'UserForm'
+      Card.create! name: "JoeForm", type: 'UserForm'
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.